src/main/java/com/thevoxelbox/voxelsniper/brush/type/entity/EntityBrush.java

```diff
@@ -4,43 +4,54 @@
 import com.sk89q.worldedit.EditSession;
 import com.sk89q.worldedit.bukkit.BukkitAdapter;
 import com.sk89q.worldedit.math.BlockVector3;
+import com.sk89q.worldedit.world.entity.EntityType;
+import com.sk89q.worldedit.world.entity.EntityTypes;
 import com.thevoxelbox.voxelsniper.brush.type.AbstractBrush;
 import com.thevoxelbox.voxelsniper.sniper.snipe.Snipe;
 import com.thevoxelbox.voxelsniper.sniper.snipe.message.SnipeMessenger;
 import com.thevoxelbox.voxelsniper.sniper.toolkit.ToolkitProperties;
 import org.bukkit.ChatColor;
 import org.bukkit.World;
-import org.bukkit.entity.Entity;
-import org.bukkit.entity.EntityType;
 
-import java.util.Arrays;
+import java.util.List;
 import java.util.stream.Collectors;
 
 public class EntityBrush extends AbstractBrush {
 
-    private EntityType entityType = EntityType.ZOMBIE;
+    private static final List<String> ENTITIES = EntityType.REGISTRY.values().stream()
+            .map(com.sk89q.worldedit.world.entity.EntityType::getId)
+            .collect(Collectors.toList());
+
+    private EntityType entityType = EntityTypes.ZOMBIE;
 
-    @SuppressWarnings("deprecation")
     @Override
     public void handleCommand(String[] parameters, Snipe snipe) {
         SnipeMessenger messenger = snipe.createMessenger();
-        if (parameters[1].equalsIgnoreCase("info")) {
-            messenger.sendMessage(ChatColor.BLUE + "The available entity types are as follows:");
-            String names = Arrays.stream(EntityType.values())
-                    .map(currentEntity -> ChatColor.AQUA + " | " + ChatColor.DARK_GREEN + currentEntity.getName())
-                    .collect(Collectors.joining("", "", ChatColor.AQUA + " |"));
-            messenger.sendMessage(names);
+        if (parameters[0].equalsIgnoreCase("info")) {
```

Comment:
Might be worth to have a `list` argument like for the tree generator listing down all possible entities in the chat.

Reply:
Isn't tab-completion enough? VS previously sent a list, a huge list in the chat, kinda annoying imo.
_Brushes no longer print all possible types._

Reply:
Having a list allows you to view everything at a glance. This doesn't need to be an _or_. We can have a list args and tab completion.